Stratis.Bitcoin.Features.Wallet/WalletFeePolicy.cs

```diff
@@ -4,13 +4,13 @@
 using Stratis.Bitcoin.Features.MemoryPool.Fee;
 using Stratis.Bitcoin.Utilities;
 using System;
-using System.Threading.Tasks;
 
 namespace Stratis.Bitcoin.Features.Wallet
 {
     public interface IWalletFeePolicy
     {
-        Task Initialize();
+        void Start();
```

Comment:
We could consider creating an interface 
```
IStratable
{
   Strat()
   Stop()
}
```

Reply:
That was where I was heading ;) But lets do this refactor in a separate task? I want to standardize all of the feature services that have Start and Stop methods.

Reply:
Create an issue (and task on the board) so we have it in the back log.